analyses/npl/05.5_station_correlation_viz.md

```diff
@@ -0,0 +1,238 @@
+```python
+from pathlib import Path
+import os
+import sys
+from collections import Counter
+
+import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+import numpy as np
+from scipy.signal import correlate
+from scipy.interpolate import interp1d
+import xarray as xr
+
+path_mod = f"{Path(os.path.dirname(os.path.realpath(''))).parents[0]}/"
+sys.path.append(path_mod)
+
+from src.indicators.flooding.glofas import utils, glofas
```

Comment:
I don't think `glofas` is used (sorry, I think it's my own mistake which you inherited)